### ISSUE TITLE
Install script crashes during "Generating SSL certificates"

### ISSUE THREAD

**Issue by Jeroen96** [lamassu/lamassu-server#95]
Hi,

I am trying to setup lamassu admin/server on my computer. I want to use this installation locally so that i can test my application. However installing everyting via the script mentioned in the next-gen admin setup post doesnt't work very well. I have tried it with a clean ubuntu 16 desktop install within virtualbox and a clean install of ubuntu 16 server within virtualbox and vmware. I have managed to get it up until "Generating SSL certificates..." where it crashes. The last error that i am getting is: "Error Loading request extension section SAN". Is anyone able to help me with this?

![screenshot_1](https://user-images.githubusercontent.com/10827065/35809828-0d8acf06-0a8a-11e8-98be-888849d7e25a.png)

**Comment by naconner** [comment]
The install script is written with a DigitalOcean VPS in mind. It won't install well outside of that environment without modification.

You may configure a VPS with the instructions here: https://support.lamassu.is/hc/en-us/articles/115001203791-Setting-up-the-Admin-Next-gen-

And a local machine here: https://github.com/lamassu/lamassu-machine/wiki/Running-v3-locally-on-Debian-or-Ubuntu

We can provide support to machine owners on our Service Level Agreement.

**Comment by joshmh** [comment]
@Jeroen96, quick update on this: We are preparing an install instructions document for local dev machines. Should be pushed today. I'll update here.

**Comment by Jeroen96** [comment]
@naconner I have looked into the local machine instructions you mentioned. Those instructions work up until the part you need to pair your machine. I have downloaded the admin git repo and made it show the login screen by fixing a couple of errors, however since i don't have a user, i'm unable to login.

@joshmh Those install instructions couldn't be timed better! I'l wait for your response.

**Comment by naconner** [comment]
@Jeroen96 The lamassu-admin repo is for our prior platform. The current admin is contained within lamassu-server@v5.

**Comment by joshmh** [comment]
@Jeroen96 try this: https://github.com/lamassu/lamassu-server/blob/v5/INSTALL.md

**Comment by Jeroen96** [comment]
@joshmh  Thx for the guides. I'm looking into them right now, but i get stuck once again during the "Generating SSL certificates" proces. It's the same error as in my first post. Do i need to add extra information in files like /etc/ssl/private/openssl.cnf or not?
![image](https://user-images.githubusercontent.com/10827065/35850573-986563f2-0b25-11e8-9b33-1275cd16c9eb.png)

**Comment by joshmh** [comment]
It looks like you're not using the latest version of cert-gen:

**Comment by Jeroen96** [comment]
@joshmh That could be possible yea. I downloaded the latest release (5.7.8) from github and didn't clone the master. I will try that.

**Comment by Jeroen96** [comment]
@joshmh I got everything up and running and my machine paired correctly to the admin. Thank you for that. I noticed a couple of things however:

1. Ether gives an error in the lamassu server with "No balance for ETH yet", wich makes the admin crash when you go to the funding tab.
2. When i click on a coin on the machine, it shows the  scan your QR code for couple of seconds before going back to the main menu, thus im unable to login. I do remember in the old setup you had to pass a wallet with a flag. Is this still the current way?
3. In which way is it a requirement that i have to run the admin with **Localhost**:8070. My idea was to run the server in the virtual machine whilst developing under Windows. Can i edit this somewere?
